etna/metrics/metrics.py

```diff
@@ -272,4 +273,33 @@ def greater_is_better(self) -> None:
         return None
 
 
-__all__ = ["MAE", "MSE", "RMSE", "R2", "MSLE", "MAPE", "SMAPE", "MedAE", "Sign"]
+class MaxDeviation(Metric):
+    """Max Deviation metric with multi-segment computation support.
+
+    .. math::
+        MaxDeviation(y\_true, y\_pred) = \\max | y_j |, where \\, y_j = \\sum_{i=1}^{n}{y\_pred_i - y\_true_i}
```

Comment:
I'm not sure about it, but shouldn't it be 
```suggestion
        MaxDeviation(y\_true, y\_pred) = \\max_{j} | y_j |, where \\, y_j = \\sum_{i=1}^{j}{y\_pred_i - y\_true_i}
```